src/components/textbox/textbox.component.js

```diff
@@ -95,14 +97,18 @@ const Textbox = ({
           id={id}
           reverse={reverse}
           isOptional={isOptional}
-          useValidationIcon={validationOnLabel}
+          useValidationIcon={!newValidationDesign && validationOnLabel}
           adaptiveLabelBreakpoint={adaptiveLabelBreakpoint}
           isRequired={required}
           data-component={dataComponent}
           data-role={dataRole}
           data-element={dataElement}
           {...filterStyledSystemMarginProps(props)}
         >
+          {props.hintText && <StyledHintText>{props.hintText}</StyledHintText>}
```

Comment:
I believe this is just the new design for `fieldHelp` so that's the text that should be displayed here. We don't need the new `hintText` prop at all

Reply:
As the rest of the new design stuff, this should only be able to render if `newValidationDesign` is true, and should replace the old `fieldHelp` stuff